src/analytics/api/views/data.py

```
@@ -53,89 +62,80 @@
+        frequency = self._get_valid_option(
+            json_data.get("frequency"), valid_options["frequencies"]
         )
-        output_format = (
-            f"{json_data.get('outputFormat', valid_output_formats[0])}".lower()
+        download_type = self._get_valid_option(
+            json_data.get("downloadType"), valid_options["download_types"]
         )
+        output_format = self._get_valid_option(
+            json_data.get("outputFormat"), valid_options["output_formats"]
+        )
+        data_type = self._get_valid_option(
```

Ruff

Local variable `data_type` is assigned to but never used

Remove assignment to unused variable `data_type`

(F841)